packages/bpk-component-scrollable-calendar/examples.js

```diff
@@ -96,9 +96,9 @@ const DefaultExample = () => (
   />
 );
 
-const WeekStartsOnSeven = () => (
+const WeekStartsOnSix = () => (
   <ScrollableCal
-    weekStartsOn={7}
+    weekStartsOn={6}
```

Comment:
`date-fns` now throws an error if you use a number not between 0-6 for this, so I changed it.